samples/Wolfram Language/MiscCalculations.nb

```diff
@@ -1,232 +1,232 @@
-(* Content-type: application/vnd.wolfram.mathematica *)

```

Comment:
Why was this file changed?

Reply:
The simple answer is that it wasn't, in terms of user-defined source code. 

The longer answer is that Mathematica's notebooks are generated from user input (code, text), but also contain output and a lot of metadata. The changes we see are related to differences in how the application compressed representations of images, the size of the window on which the application was run, and the time at which the notebook and each of its input cells were last changed. MiscCalculations.nb contains only 7 lines of user-defined Wolfram Language code, and 225 lines of stuff that's generated by Mathematica. 

We can safely revert these changes, which were likely caused by re-evaluating the notebook.